src/components/CardHeader.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { useState, useEffect } from 'react';
+
+function CardHeader() {
+  const months = [
+    'JAN',
+    'FEB',
+    'MAR',
+    'APR',
+    'MAY',
+    'JUNE',
+    'JULY',
+    'AUG',
+    'SEPT',
+    'OCT',
+    'NOV',
+    'DEC',
+  ];
+  const days = [
+    'SUNDAY',
+    'MONDAY',
+    'TUESDAY',
+    'WEDNESDAY',
+    'THURSDAY',
+    'FRIDAY',
+    'SATURDAY',
+  ];
+
+  const now = new Date();
+  const [date, setDate] = useState(now.getDate());
+  const [month, setMonth] = useState(now.getMonth());
+  const [day, setDay] = useState(now.getDay());
+  const [year, setYear] = useState(now.getFullYear());
+
+  // Update Date,Month and Day every second
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setDate(new Date().getDate());
+      setMonth(new Date().getMonth());
+      setDay(new Date().getDay());
+      setYear(new Date().getFullYear());
+    }, 1000);
```

Comment:
Since we are only displaying the date, then `useEffect` should run every hour instead of every second to improve performance.